include/pokemon.h

```diff
@@ -451,44 +451,40 @@ struct MoveInfo
     const u8 *name;
     const u8 *description;
     u16 effect;
-    u8 power;
-    u8 type:5;
-    u8 category:3;
-
+    u16 type:5;
+    u16 category:2;
+    u16 power:9; // up to 511
     u16 accuracy:7;
-    u16 recoil:7;
-    u16 criticalHitStage:2;
-    u8 padding:6; // coming soon...
-    u8 numAdditionalEffects:2; // limited to 3 - don't want to get too crazy
+    u16 target:9;
     u8 pp;
-
-    u16 target;
-    s8 priority;
     union {
         u8 effect;
         u8 powerOverride;
     } zMove;
 
+    s32 priority:4;
+    u32 recoil:7;
+    u32 strikeCount:4; // Max 15 hits. Defaults to 1 if not set. May apply its effect on each hit.
+    u32 criticalHitStage:2;
+    u32 alwaysCriticalHit:1;
+    // 14 bits left to complete this word - continues into flags
```

Comment:
Add this padding as a field.

Reply:
It's not padding though. Like it says, it just continues into flags. It's just a note for anyone who would like to add an extra field in that position.

Reply:
You could add a space to delimit where the word end instead (or a comment)